jib-core/src/main/java/com/google/cloud/tools/jib/http/Authorization.java

```diff
@@ -27,10 +29,37 @@
  */
 public class Authorization {
 
+  /**
+   * @param token the token
+   * @return an {@link Authorization} with a {@code Bearer} token
+   */
+  public static Authorization withBearerToken(String token) {
```

Comment:
I think these can be in the style of `fromX` instead of `withX` if we're moving them from an `Authorizations` class.